pydoc/quickstart.rst

```diff
@@ -49,98 +92,28 @@ In another terminal window, launch an interactive Python 3 session, such as ``ip
    ...
    connected to tcp://node01:5555
```

Comment:
in this block we do
```python
   >>> import arkouda as ak
   >>> default way to connect is
   >>> ak.connect(connect_url='tcp://node01:5555')
   ...
   connected to tcp://node01:5555
```

For the second line,  the `>>>` makes it seem like a command. I think it would look better if we changed the line to a comment instead 
```python
   >>> import arkouda as ak
   # default way to connect is
   >>> ak.connect(connect_url='tcp://node01:5555')
   ...
   connected to tcp://node01:5555
```

Reply:
Definitely agree. Updated.